pkg/api/configsync/v1beta1/helmconfig.go

```diff
@@ -59,31 +59,16 @@ type HelmBase struct {
 
 	// values to use instead of default values that accompany the chart. Format
 	// values the same as default values.yaml. These values will take precedence if
-	// used in conjunction with valuesFileSources. How to handle multiple valuesFiles is
-	// determined by `valuesFileApplyStrategy`.
+	// used in conjunction with valuesFileSources.
 	// +optional
 	Values *apiextensionsv1.JSON `json:"values,omitempty"`
 
 	// valuesFileSources holds references to objects in the cluster that repremakesent
 	// values to use instead of default values that accompany the chart. Currently,
 	// only ConfigMaps are supported. Objects listed later will take precedence.
-	// How to handle multiple valuesFiles is determined by `valuesFileApplyStrategy`.
 	// +optional
 	ValuesFileSources []ValuesFileSources `json:"valuesFileSources,omitempty"`
 
-	// valuesFileApplyStrategy specifies the strategy for handling multiple valueFiles. It
-	// refers to how valuesFiles are applied if multiple valuesFile define the
-	// same key. Can be 'override' or 'listConcatenate'.
-	// 'override' (default) results in the duplicated keys in later files to
-	// override the value from earlier files. This is equivalent to passing in
-	// multiple valuesFiles to Helm CLI.
```

Comment:
Move the part about the override/default behavior to `valuesFileSources` (but don't mention override).

Reply:
done